docker/docker-compose.yaml

```diff
@@ -75,8 +75,11 @@ services:
         ports: []
         environment:
             <<: *php-env
-        command: ["-f"]
-        entrypoint: "/usr/sbin/crond"
+#        command: ["-f"]
+#        command: ["-f", "-d", "3", "-l", "3"]
+#        entrypoint: "/usr/sbin/crond"
+        command: ["-c exit"]
+        entrypoint: "sh"
```

Comment:
This change in the `cron` container might result in it not running as expected. The change from a cron daemon command to `sh -c exit` means the container will exit immediately after starting, which is not usually the desired behavior for a cron service. Please confirm if this is the intentional behavior.